main.go

```diff
@@ -21,27 +22,31 @@ import (
 	"strings"
 	"time"
 
-	"github.com/minio/minio-go/v6"
-	"github.com/minio/minio-go/v6/pkg/credentials"
-	"github.com/minio/minio-go/v6/pkg/s3utils"
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
 	log "github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
 const (
 	backupBaseDir         = "/backup"
-	backupRetries         = 4
+	defaultBackupRetries  = 4
```

Comment:
Just curious why the default is 4 retries and s3 is 3 retries? If you used the same var, say retries=3 for both you could reduce the code a little